LayoutTests/fast/images/decode-static-svg-image-resolve.html

```diff
@@ -0,0 +1,37 @@
+<style>
+/* This tests passes if two green rectangles are drawn. */
```

Comment:
This comment looks like a debugging comment. Usually we put this message in `<p>...</p>` or `<h3>...</h3>` to make it easy to know what is expected from this file once it is displayed.  Some tests put this message in `<title>...</title>`.

Reply:
Added `<p>...</p>`.